cmd/collect_signals/README.md

```diff
@@ -0,0 +1,205 @@
+# Signal Collector
+
+This tool is used to collect signal data for a set of project repositories for
+generating a criticality score.
+
+The input of this tool could by the output of the `enumerate_github` tool.
+
+The output of this tool is used as an input for the `scorer` tool, or for input
+into a data analysis system such as BigQuery. 
+
+## Example
+
+```shell
+$ export GITHUB_TOKEN=ghp_x  # Personal Access Token Goes Here 
+$ export ...
+$ collect_signals \
+    -workers=1 \
+    github_projects.txt \
+    signals.csv
+```
+
+## Install
+
+```shell
+$ go install github.com/ossf/criticality_score/cmd/collect_signals
+```
+
+## Usage
+
+```shell
+$ collect_signals [FLAGS]... IN_FILE... OUT_FILE
+```
+
+Project repository URLs are read from each `IN_FILE` specified. If `-` is passed
+in as an `IN_FILE` URLs will read from STDIN, along with any other files specified.
+
+Results are written in CSV format to `OUT_FILE`. If `OUT_FILE` is `-` the
+results will be written to STDOUT.
+
+`FLAGS` are optional. See below for documentation.
+
+### Authentication
+
+`collect_signals` requires authentication to GitHub, and optionally Google Cloud Platform to run.
+
+#### GitHub Authentication
+
+A comma delimited environment variable with one or more GitHub Personal Access
+Tokens must be set
+
+Supported environment variables are `GITHUB_AUTH_TOKEN`, `GITHUB_TOKEN`, 
+`GH_TOKEN`, or `GH_AUTH_TOKEN`.
+
+Example:
+
+```shell
+$ export GITHUB_TOKEN=ghp_abc,ghp_123
+```
+
+#### GCP Authentication
+
+BigQuery access requires the "BigQuery User" (`roles/bigquery.user`) role added
+to the account used, or be an "Owner".
+
+##### Option 1: `gcloud login`
+
+This option is useful during development. Run `gcloud login` to login to GCP.
```

Comment:
Do we need `--update-adc` here for the client libraries to work?

Reply:
Added. Thanks.